ui/src/main/webapp/home/actions.js

```diff
@@ -0,0 +1,37 @@
+import { poll } from 'redux-polling'
+import { get, post } from 'redux-fetch'
+
+// actions
+
+export const setConfigs = (id, value) => ({type: 'HOME/SET_CONFIGS', id, value})
+export const beginDelete = (id, servicePid) => ({type: 'HOME/BEGIN_SUBMITTING_CONFIGS', id, servicePid})
+
+// async actions
+
+export const retrieve = (id) => async (dispatch) => {
+  const res = await dispatch(get('/admin/beta/config/configurations/' + id))
+  const json = await res.json()
+
+  if (res.status === 200) {
+    dispatch(setConfigs(id, json))
+  }
+}
+
+export const refresh = poll('home', () => (dispatch) =>
```

Comment:
I'd prefer to wait on this until we have multiple home pages for security, federation and a home dashboard before tackling this. I'm more than positive we'll just end up rewriting whatever changes we make right now.